setup.py

```diff
@@ -15,6 +15,28 @@
 LONG_DESCRIPTION_FILE = os.path.join(TOPDIR, 'README.rst')
 VERSION_FILE = os.path.join(TOPDIR, 'shoop', '_version.py')
 
+# Release instructions
```

Comment:
I don't think these belong in a comment in setup.py. Can you make a Real, Actual Document out of these -- maybe .rst in `doc/`, even?